bucket/cactusviewer.json

```diff
@@ -12,6 +12,9 @@
       }
     }
   },
+  "bin": [
+    "cactusviewer.exe"
+  ],
```

Comment:
https://github.com/ScoopInstaller/.github/blob/43372d6e0e9f00375f7fe7a2226da9254bfce5e7/.github/CONTRIBUTING.md?plain=1#L96

>    * If an array contains only one item, write it as a string.

Reply:
Thanks. I also missed tab width of 4 spaces. Both should be possible to check for programatically.